### ISSUE TITLE
[Screen Reader-Intelligent Support Experience-Rate your experience]: Screen readers (NVDA and JAWS) are not narrating the name and state for the star rating controls present in 'Rate your experience' message section.

### ISSUE THREAD

**Issue by Amit8527510735** [microsoft/BotFramework-WebChat#3581]
**User Impact:** 
Screen readers users will face difficulty while navigating over the star rating controls if NVDA and JAWS doesn't narrate the name and state. Hence users might not be able to understand the functionality and the purpose of these controls.

**Test Environment:**
OS : Windows 10 2004 (OS Build 19041.450)
Browser: New Edge 84.0.522.63, Chrome 84.0.4147.135
URL: https://servicesuat.microsoft.com/#/time/timeentry
 Screen readers: NVDA 2020.2, JAWS 2020.2004.13.400

**Repro Steps:**
Step 1: Open the above URL in Edge/Chrome browser with valid credentials and turn on NVDA/JAWS.
Step 2: Say 'Hi' to Iris chat bot.
Step 3: 'Would you like me to create a support ticket for you?' response comes back.
Step 4: Choose 'No' to reply back.
Step 5: Reply with 'No' for 'Is there anything else that I can help you with?' response from chat bot.
Step 6: Enable browse/cursor modes and navigate over the star rating buttons of 'Rate your experience!' message using arrow keys.
Step 7: Verify whether screen reader (NVDA/JAWS) is announcing the name and state 
for the star rating controls or not.

**Actual:**
Screen readers (NVDA and JAWS) are not narrating the name and state for the star rating controls present in 'Rate your experience' message section.
NVDA is announcing as "undefined button" and JAWS is announcing as "undefined attachment"

**Expected:**
Name and state should be defined for the star rating controls present in 'Rate your experience' message section.
Ex: Screen readers should announce as "1 star button selected/not selected"

**MAS Reference Link:**
[MAS 4.1.2 – Name, Role, Value ](https://microsoft.sharepoint.com/:w:/s/accessibility/EXTXYbbQYHdCoSFz-Ybr7nYBBral6s8qIHeXupQiG7jixw?e=GkZwHJ)
 
**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
https://www.w3.org/WAI/WCAG21/Techniques/general/G108

![MAS 4 1 2_Star rating controls](https://user-images.githubusercontent.com/53263029/97972298-51fcb200-1dea-11eb-895e-ee88d83fa071.png)

- [ ]

**Comment by compulim** [comment]
This is a narration inside Adaptive Cards, https://github.com/microsoft/adaptivecards.

@anusharavindrar please send this issue to the Adaptive Cards team instead. Thanks!
